.github/workflows/golangci-lint.yml

```diff
@@ -13,22 +13,16 @@ permissions:
   contents: read
 
 jobs:
-  build:
-    name: Build
+  lint:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - name: Setup Go
-        uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
-        with:
-          go-version-file: go.mod
-          cache: false
-      - run: go mod download
-      - run: go build -v .
+        uses: ./.github/actions/setup-go
+      - name: Download Go Modules
+        run: go mod download
+      - name: Build
+        run: go build -v .
       - name: Run Linters
-        uses: golangci/golangci-lint-action@1481404843c368bc19ca9406f87d6e0fc97bdcfd # v7.0.0
-        with:
-          version: latest
-          args: --issues-exit-code=1 --timeout 10m
-          only-new-issues: true
+        uses: ./.github/actions/golangci-lint
```

Comment:
Should we set `only-new-issues` here? As the action defaults to `false`

Reply:
Updated along with the addition of a scheduled run each Sunday at midnight UTC.